stable/vpa/README.md

```diff
@@ -135,3 +175,7 @@ recommender:
 | admissionController.tolerations | list | `[]` |  |
 | admissionController.affinity | object | `{}` |  |
 | tests.securityContext | object | `{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"readOnlyRootFilesystem":true,"runAsNonRoot":true,"runAsUser":10324}` | The security context for the containers run as helm hook tests |
+| tests.image.repository | string | `"bitnami/kubectl"` | An image used for testing containing bash, cat and kubectl |
+| tests.image.tag | string | `""` | An image tag for the tests image |
+| tests.image.pullPolicy | string | `"Always"` | The pull policy for the tests image. |
+| metrics-server.enabled | bool | `true` |  |
```

Comment:
This either needs to default to "false" with a change to ci/test-values, or be moved to a pre-test script for installation during tests. Most environments will not need this, and it will break existing users of Goldilocks, which I would like to avoid.

https://github.com/FairwindsOps/charts/blob/master/scripts/pre-test-script-runner.sh

Reply:
I would opt for keeping it as a dependency and disabling it by default. I think, it clarifies, that VPA has the metrics-server as a dependency.
Thank you for your comment, as I was not aware of the implications of this change.